main.js

```diff
@@ -21,4 +21,80 @@ closebutton.addEventListener('click', closeMenu);
 const listItem = document.querySelectorAll('.menu-btn');
 listItem.forEach((item) => {
   item.addEventListener('click', closeMenu);
-});
\ No newline at end of file
+});
+
+const projectInfo = {
+  work1: {
+    name: 'Multi-Post Stories',
+    image: 'img/work_img/Portfolio.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    live: 'https://chuanqizhu0820.github.io/My-Portfolio/',
+    source: 'https://github.com/chuanqizhu0820/My-Portfolio',
+  },
+  work2: {
+    name: 'Multi-Post Stories',
+    image: 'img/work_img/Portfolio1.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    live: 'https://chuanqizhu0820.github.io/My-Portfolio/',
+    source: 'https://github.com/chuanqizhu0820/My-Portfolio',
+  },
+  work3: {
+    name: 'Multi-Post Stories',
+    image: 'img/work_img/Portfolio2.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    live: 'https://chuanqizhu0820.github.io/My-Portfolio/',
+    source: 'https://github.com/chuanqizhu0820/My-Portfolio',
+  },
+  work4: {
+    name: 'Multi-Post Stories',
+    image: 'img/work_img/Portfolio3.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    live: 'https://chuanqizhu0820.github.io/My-Portfolio/',
+    source: 'https://github.com/chuanqizhu0820/My-Portfolio',
+  },
+};
+
+function openWindow() {
+  const popupwindow = document.querySelector('#popup-window');
+  const workid = this.id;
+
+  const protitle = document.querySelector('#windowtitle');
+  protitle.textContent = projectInfo[workid].name;
+
+  const proimg = document.querySelector('#windowimg');
+  proimg.src = projectInfo[workid].image;
+
+  const protech = document.querySelectorAll('#windowtech li');
+  protech.forEach((item, i) => {
```

Comment:
- [ ]  As per Figma design the project title and popup close icon should be on extreme end of the popup. Kindly check the screenshot below and try to make the required changes for both mobile views as well as desktop views.

- [ ] On the Desktop view, The project description should be in the center. Kindly compare the below image and fix the issue.

Yours
![image](https://user-images.githubusercontent.com/53019916/131879034-5db99e14-fa8e-49fc-9c18-b4310e39a0d2.png)


Figma Design
![image](https://user-images.githubusercontent.com/53019916/131879135-a332acee-e346-44fe-97ae-a4991c8078f8.png)

Your pop up on mobile view
![image](https://user-images.githubusercontent.com/53019916/131880052-b802e4e9-e842-43da-ad7a-129102a18347.png)

Reply:
Hello, @hemant-soni-vst-au4. I made the changes according to your suggestions, but the second reviewer does not approve it. What should I do?